pypeit/core/wavecal/autoid.py

```diff
@@ -854,8 +854,46 @@ def reidentify(spec, spec_arxiv_in, wave_soln_arxiv_in, line_list, nreid_min, de
     return detections, spec_cont_sub, patt_dict_slit
 
 
+def measure_fwhm(spec, binspectral):
+    """
+    Measure the arc lines FWHM, i.e, approximate spectral resolution
+
+    Args:
+        spec (`numpy.ndarray`_):
+            Arc spectrum from a single slit
+        binspectral (:obj:`int`):
+          Binning of the input arc in the spectral dimension
+
+    Returns:
+        :obj:`float`: Measured FWHM in pixels
```

Comment:
Got myself confused on binning when I was thinking about this before.

Is the FWHM reported in the binning of the arc, native pixels, or the binning of the science?   Let's specify in the docs (and in `WaveFit`).  Thx!

Reply:
Done